### PR TITLE
add inverseRobot task solution

### PR DESCRIPTION
**Pull request by opiquem**
add inverseRobot task solution

### CODE REVIEW

File: src/inverseRobot.js [hunk 1]
```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const reversedRobot = {};
+
+  for (const key in robot) {
+    if (Object.keys(reversedRobot).includes(robot[key])) {
+      return null;
+    }
```
Comment (ruslanpashkov) [inline src/inverseRobot.js:15]:
```suggestion
    if (reversedRobot[robot[key]]) {
      return null;
    }
```

**Review by ruslanpashkov — APPROVED** [review]

File: src/inverseRobot.js [hunk 2]
```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const reversedRobot = {};
+
+  for (const key in robot) {
+    if (Object.keys(reversedRobot).includes(robot[key])) {
+      return null;
```
Comment (YeMyr) [inline src/inverseRobot.js:14]: nice

**Review by YeMyr — APPROVED** [review]

**Review by YeMyr — APPROVED** [review]

File: src/inverseRobot.js [hunk 3]
```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const reversedRobot = {};
+
+  for (const key in robot) {
+    if (Object.keys(reversedRobot).includes(robot[key])) {
```
Comment (Anna-Tkachenko) [inline src/inverseRobot.js:13]: `Object.keys()` use loop inside for creating an array of keys. Let's try to check the existence of the key without using extra loops

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

**Review by Anna-Tkachenko — APPROVED** [review]
⭐️

**Pull request closed without merging**